examples/md-flexible/input/testTimePerTraversal/TestTimePerTraversal1.yaml

```diff
@@ -0,0 +1,22 @@
+# Generated by:
+# examples/md-flexible/md-flexible 
+selector-strategy                :  Fastest-Absolute-Value
+functor                          :  Lennard-Jones (12-6)
+cutoff                           :  2
+deltaT                           :  0
+# 138 tuning iterations + 50 to observe choses configuration
+iterations                       :  50
+periodic-boundaries              :  false
+Objects:
+  CubeGrid:
+    0:
+      particles-per-dimension    :  [10, 10, 10]
```

Comment:
I think it would be simpler, to only have one yaml file per test type and then overwrite the parameter that is scaled with command line arguments.